{{cookiecutter.project_slug}}/docker-compose.yml

```
@@ -112,7 +112,7 @@ services:
     # You also have to change the flower command
   
   flower:
-    image: mher/flower
+    image: mher/flower:0.9.4
```

As of at least today, the more recent tag for flower `0.9.7` works for this as well. Maybe worth bumping the version number. 
```suggestion
    image: mher/flower:0.9.7
```